torchci/test/labelBot.test.ts

```diff
@@ -0,0 +1,159 @@
+import nock from "nock";
+import * as probot from "probot";
+import * as utils from "./utils";
+import labelBot from "lib/bot/labelBot";
+
+nock.disableNetConnect();
+
+describe("label-bot", () => {
+  let probot: probot.Probot;
+
+  const existingRepoLabelsResponse = [
+    {
+      id: 1301447942,
+      node_id: "MDU6TGFiZWwxMzAxNDQ3OTQy",
+      url: "https://api.github.com/repos/pytorch/pytorch/labels/enhancement",
+      name: "enhancement",
+      color: "ffc6f4",
+      default: true,
+      description:
+        "Not as big of a feature, but technically not a bug. Should be easy to fix",
+    },
+    {
+      id: 1150417147,
+      node_id: "MDU6TGFiZWwxMTUwNDE3MTQ3",
+      url: "https://api.github.com/repos/pytorch/pytorch/labels/good%20first%20issue",
+      name: "good first issue",
+      color: "34c182",
+      default: true,
+      description: "",
+    },
```

Comment:
Can we move this one to fixture? (I'm surprised we already don't have one)

Reply:
Correct me if I'm wrong:
From what I see that fixtures are used to trigger the bot events. I didn't find any place where fixtures are used to mock response from API  as we do here: `.reply(200, existingRepoLabelsResponse)`. 

JSON in fixtures and this object have different structure. This is just a small part of what can be in the fixture file.
Do we still want to put this response in fixtures?

Reply:
Suppose ones `fixture/known_labels.json' looks as follows:
```
[
    {
      "id": 1301447942,
      "node_id": "MDU6TGFiZWwxMzAxNDQ3OTQy",
      "url": "https://api.github.com/repos/pytorch/pytorch/labels/enhancement",
      "name": "enhancement",
      "color": "ffc6f4",
      "default": true,
      "description":
        "Not as big of a feature, but technically not a bug. Should be easy to fix"
    },
    {
      "id": 1150417147,
      "node_id": "MDU6TGFiZWwxMTUwNDE3MTQ3",
      "url": "https://api.github.com/repos/pytorch/pytorch/labels/good%20first%20issue",
      "name": "good first issue",
      "color": "34c182",
      "default": true,
      "description": ""
    }
]
```

then `const existingRepoLabelsResponse = require("./fixtures/known_labels.json");` would be equivalent to the code you have right now, wouldn't it?

Reply:
Created the fixture file